tests/Colder.Tests/InMemoryDistributedLockTest.cs

```diff
@@ -36,10 +37,10 @@ public async Task Test()
             {
                 tasks.Add(Task.Factory.StartNew(async () =>
                 {
-                    for (int j = 0; j < 10000; j++)
+                    for (int j = 0; j < 100; j++)
                     {
                         using var _ = await theLock.Lock(key);
-                        num++;
+                        Interlocked.Increment(ref num);
```

Comment:
Interlocked is thread safe,but this code is to test the distributedlock is work fine or not